domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/TabShowConditionValidatorImplTest.java

```diff
@@ -144,16 +150,76 @@ public void TabFieldShowCondition_returnsNoValidationErrorsOnSuccessWhenReferenc
         assertThat(result.isValid(), is(true));
     }
 
+    @Test
+    public void TabFieldShowCondition_shouldValidateShowConditionForCustomComplexField() throws InvalidShowConditionException {
```

Comment:
I know this was not introduced inside this PR but would be nice to have TabFieldShowCondition and TabShowCondition as two separate subclasses in junit 5 nested tests style.